arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -534,9 +534,9 @@ public void traceEndTransaction(
 
     // TODO: add the following resolution this.defers.resolvePostRollback(this, ...
 
-    txStack.current().completeLineaTransaction(this, isSuccessful, logs, selfDestructs);
-
+    txStack.current().completeLineaTransaction(this, world, isSuccessful, logs, selfDestructs);
     defers.resolveAtEndTransaction(this, world, tx, isSuccessful);
+    defers.resolveAfterTransactionFinalization(this, world);
```

Comment:
The only items that are in the `resolveAfterTransactionFinalization` defer registry are the "account wiping rows" of successful `SELFDESTRUCT`'s.